assets/src/stories-editor/components/font-family-picker/status.js

```diff
@@ -0,0 +1,90 @@
+
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+
+/**
+ * WordPress dependencies
+ */
+import { Component } from '@wordpress/element';
+
+export default class Status extends Component {
+	constructor( props ) {
+		super( props );
+		this.state = {
+			bump: false,
+		};
+	}
+
+	componentWillReceiveProps( { queryLength } ) {
+		const hasChanged = queryLength !== this.props.queryLength;
+		if ( hasChanged ) {
+			this.setState( ( { bump } ) => ( { bump: ! bump } ) );
+		}
+	}
+
+	render() {
+		const {
+			length,
+			queryLength,
+			minQueryLength,
+			selectedOption,
+			selectedOptionIndex,
+			tQueryTooShort,
+			tNoResults,
+			tSelectedOption,
+			tResults,
+		} = this.props;
+		const { bump } = this.state;
+
+		const queryTooShort = queryLength < minQueryLength;
+		const noResults = length === 0;
+
+		const contentSelectedOption = selectedOption ?
+			tSelectedOption( selectedOption, length, selectedOptionIndex ) :
+			'';
+
+		let content = null;
+		if ( queryTooShort ) {
+			content = tQueryTooShort( minQueryLength );
+		} else if ( noResults ) {
+			content = tNoResults();
+		} else {
+			content = tResults( length, contentSelectedOption );
+		}
+
+		return <div
+			aria-atomic="true"
+			aria-live="polite"
+			role="status"
+			style={ {
+				border: '0',
+				clip: 'rect(0 0 0 0)',
+				height: '1px',
+				marginBottom: '-1px',
+				marginRight: '-1px',
+				overflow: 'hidden',
+				padding: '0',
+				position: 'absolute',
+				whiteSpace: 'nowrap',
+				width: '1px',
+			} }
+		>
+			{ content }
+			<span>{ bump ? ',' : ',,' }</span>
```

Comment:
What does this `bump` do exactly?

Reply:
Copied from upstream: https://github.com/alphagov/accessible-autocomplete/blob/baee8582ae28ec499289eded99053e0c27ebf33d/src/status.js#L77